tests/jupyterhub_api/helper.py

```diff
@@ -0,0 +1,56 @@
+import os
+import glob
+import sys
+import subprocess
+from test_check_api import  CheckAPI
+from test_stop_session import StopSession
+from test_create_session import CreateSession
+from test_check_session import CheckSession
+
+
+def run_jupyterhub_api(tasks):
```

Comment:
Here I have the same comment I gave in the other PR. This is not modular and ignoring a test is not possible. But let's put things working and then we fix it.

Reply:
I'll submit a separate PR for this.